service_info/static/less/base/ratings.less

```diff
@@ -0,0 +1,67 @@
+@import "../includes/flex.less";
+@import "../includes/mixins.less";
+
+#page-rating {
+  .captcha {
+    > .g-recaptcha > div > div {
+      margin: 0 auto;
+    }
+  }
+
+  .stars-row {
+    .flex-layout();
+    .flex-direction(row);
+    .justify-content(space-between);
+    .align-items(center);
+
+    .caption {
+      text-transform: uppercase;
+    }
+
+    .stars {
+      .flex-layout();
+      .flex-direction(row-reverse);
+      .justify-content(flex-end);
+
+      margin-top: -5px;
+    }
+  }
+
+  .spacer {
+    .flex-grow(1);
+  }
+
+  [type="radio"]+label:before,
+  [type="radio"]+label:after {
+    content: "☆";
+    color: #ffc107;
+    border: none;
+    font-size: 24px;
+  }
+
+  [type="radio"]+label:after {
```

Comment:
I don't understand the empty ``{}``.  What does that do?  Also, there's some duplicate sections (e.g., multiple ``[type="radio"]:checked+label:before {}``.  Significant because of where it appears?